lib/awful/hotkeys_popup/keys/qutebrowser.lua

```diff
@@ -0,0 +1,187 @@
+---------------------------------------------------------------------------
+--- Qutebrowser hotkeys for awful.hotkeys_widget
+--
+-- @author Simon Désaulniers sim.desaulniers@gmail.com
+-- @copyright 2017 Simon Désaulniers
+-- @module awful.hotkeys_popup.keys.qutebrowser
+---------------------------------------------------------------------------
+
+local hotkeys_popup = require("awful.hotkeys_popup.widget")
+
+local qutebrowser_rule_any = {class={"qutebrowser", "qutebrowser"}}
```

Comment:
You just need it once here.
And not `rule_any` if it should just match a single class.

I suggest using `instance = "qutebrowser"`, if that works for you.
(and not `rule_any` then)